lib/api/controller/auth.js

```diff
@@ -326,7 +365,14 @@ class AuthController extends NativeController {
    * @returns {Promise<object>}
    */
   async checkToken (request) {
-    const token = this.getBodyString(request, 'token');
+    let token = null;
```

Comment:
If you really want to initialize this variable then you can use an empty string to keep same type after assignment